src/util/feefrac.h

```diff
@@ -186,23 +190,35 @@ struct FeeFrac
     /** Compute the fee for a given size `at_size` using this object's feerate.
      *
      * This effectively corresponds to evaluating (this->fee * at_size) / this->size, with the
-     * result rounded down (even for negative feerates).
+     * result rounded towards negative infinity (if RoundDown) or towards positive infinity
+     * (if !RoundDown).
      *
      * Requires this->size > 0, at_size >= 0, and that the correct result fits in a int64_t. This
      * is guaranteed to be the case when 0 <= at_size <= this->size.
      */
+    template<bool RoundDown>
```

Comment:
Question: why did you make this a templated function instead of adding a bool parameter? Is it a compile-time optimization?

Reply:
Makes sense!